pkg/proxy/ipvs/ipset.go

```diff
@@ -65,11 +65,21 @@ const (
 	kubeNodePortLocalSetUDPComment = "Kubernetes nodeport UDP port with externalTrafficPolicy=local"
 	kubeNodePortLocalSetUDP        = "KUBE-NODE-PORT-LOCAL-UDP"
 
-	kubeNodePortSetSCTPComment = "Kubernetes nodeport SCTP port for masquerade purpose"
-	kubeNodePortSetSCTP        = "KUBE-NODE-PORT-SCTP"
+	// This ipset is no longer active but still used in previous versions.
+	// DO NOT create an ipset using this name
+	legacyKubeNodePortSetSCTPComment = "Kubernetes nodeport SCTP port for masquerade purpose"
+	legacyKubeNodePortSetSCTP        = "KUBE-NODE-PORT-SCTP"
 
-	kubeNodePortLocalSetSCTPComment = "Kubernetes nodeport SCTP port with externalTrafficPolicy=local"
-	kubeNodePortLocalSetSCTP        = "KUBE-NODE-PORT-LOCAL-SCTP"
+	// This ipset is no longer active but still used in previous versions.
+	// DO NOT create an ipset using this name
+	legacyKubeNodePortLocalSetSCTPComment = "Kubernetes nodeport SCTP port with externalTrafficPolicy=local"
+	legacyKubeNodePortLocalSetSCTP        = "KUBE-NODE-PORT-LOCAL-SCTP"
```

Comment:
Some alternatives we've considered so far:
* add logic to delete/clean up ipset at start up, but as @lbernail mentioned, this is backwards incompatible on rollback since older versions will still use type `bitmap` 
* revert https://github.com/kubernetes/kubernetes/pull/74341 but that means no SCTP support for IPVS? I don't think that's an option